igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/GitlabCiConfig.java

```diff
@@ -34,78 +38,82 @@
 import retrofit.client.OkClient;
 import retrofit.converter.JacksonConverter;
 
-import java.util.Map;
-import java.util.concurrent.TimeUnit;
-import java.util.function.Function;
-import java.util.stream.Collectors;
-
 @Configuration
 @ConditionalOnProperty("gitlab-ci.enabled")
 @EnableConfigurationProperties(GitlabCiProperties.class)
 public class GitlabCiConfig {
-    private static final Logger log = LoggerFactory.getLogger(GitlabCiConfig.class);
+  private static final Logger log = LoggerFactory.getLogger(GitlabCiConfig.class);
 
-    @Bean
-    public Map<String, GitlabCiService> gitlabCiMasters(BuildServices buildServices,
-                                                        final IgorConfigurationProperties igorConfigurationProperties,
-                                                        GitlabCiProperties gitlabCiProperties,
-                                                        ObjectMapper objectMapper) {
-        log.info("creating gitlabCiMasters");
-        Map<String, GitlabCiService> gitlabCiMasters = gitlabCiProperties.getMasters().stream()
-            .map(gitlabCiHost ->
-                gitlabCiService(igorConfigurationProperties, "gitlab-ci-" + gitlabCiHost.getName(), gitlabCiHost, objectMapper))
+  @Bean
+  public Map<String, GitlabCiService> gitlabCiMasters(
+      BuildServices buildServices,
+      final IgorConfigurationProperties igorConfigurationProperties,
+      GitlabCiProperties gitlabCiProperties,
+      ObjectMapper objectMapper) {
```

Comment:
oo this is weird formatting. should do either adam or fletch style, not both

Reply:
Too bad!